apps/aehttp/src/aehttp_api_validate.erl

```diff
@@ -127,13 +127,24 @@ populate_param(Param, Req, Validator, EndpointsMod) ->
     Name = proplists:get_value("name", Param),
     case get_param_value(In, Name, Req) of
         {ok, Value, Req1} ->
-            case prepare_param(Param, Value, Name, Validator, EndpointsMod) of
+            case prepare_param(params_priority(Param), Value, Name, Validator, EndpointsMod) of
```

Comment:
It is an implicit requirement that `default` goes first as otherwise it gets ignored :)